dockstore-webservice/src/main/resources/migrations.1.6.0.xml

```diff
@@ -235,4 +235,34 @@
                       tableName="event"/>
         <sql dbms="postgres">ALTER SEQUENCE organisation_id_seq RENAME TO organization_id_seq</sql>
     </changeSet>
+
+    <changeSet author="aduncan" id="addDisplayNameToOrganization">
+        <addColumn tableName="organization">
+            <column name="displayname" type="varchar(50 BYTE)">
+                <constraints nullable="false"/>
+            </column>
+        </addColumn>
+    </changeSet>
+    <changeSet author="aduncan" id="addDisplayNameToOrganizationUniqueConstraint">
+        <update tableName="organization">
+            <column name="displayname" valueComputed="name"/>
+            <where>displayname is null</where>
+        </update>
+        <addUniqueConstraint columnNames="displayname" constraintName="uk_orgDisplayName" tableName="organization"/>
+    </changeSet>
+
+    <changeSet author="aduncan (generated)" id="addDisplayNameToCollection">
+        <addColumn tableName="collection">
+            <column name="displayname" type="varchar(50 BYTE)">
```

Comment:
I'm not familiar with the `BYTE` in `50 BYTE` syntax... If it really means bytes, then it could potentially allow fewer than 50 characters, as 1 character is not always equal to 1 byte. Real edge case, but maybe you should just have 50 without `BYTE`, assuming that syntax works.

Reply:
This had been autogenerated by this annotation @Size(min = 3, max = 50).